extension/js/keyboardShortcuts.js

```diff
@@ -54,6 +54,9 @@ function checkKey(e) {
 
 }
 
+document.querySelector('.icon-search-active').innerHTML='"<a href=\"./search.php?search_id=active_topics\" accessKey="a" role=\"menuitem\">אקטיווע אשכולות</a>"';
```

Comment:
```suggestion
document.querySelector('.icon-search-active a').accessKey = "a"';
```

Reply:
Oh wow! I was breaking my head off how to select the anchor in the list item...

@mayim-chayim you have an extra single quote at the end.

Reply:
fixed